### PR TITLE
Plugins format fixed

### PR DESCRIPTION
**Pull request by vpavliv**
Returned old format of plugin parameters

### CODE REVIEW

**Comment by AKorpusenko** [comment]
Also added support of old API in private_message and debug_node(a9211cc252cf522f101a39e62ad3191b94e0ab41) plugins to save API compatibility.

File: libraries/plugins/account_by_key/account_by_key_plugin.cpp [hunk 1]
```diff
@@ -264,12 +265,10 @@ namespace golos {
 
             // Api Defines
             DEFINE_API(account_by_key_plugin, get_key_references) {
-                auto tmp = args.args->at(0).as<key_references_a>();
+                auto tmp = args.args->at(0).as<vector<public_key_type>>();
```
[the remark this answers is not in the document]
Comment (vpavliv) [reply libraries/plugins/account_by_key/account_by_key_plugin.cpp]: Wel ok, no problem. Its just such functions that does not do anything besides unpacking parameters and passing em to other function. So this function kinda shouldnt even know what those params do.   Just my thoughts process. I ll do as u say np.

File: libraries/plugins/block_info/plugin.cpp [hunk 2]
```diff
@@ -104,20 +104,20 @@ void plugin::plugin_impl::on_applied_block(const protocol::signed_block &b) {
...
     return db.with_read_lock([&]() {
-        return  my->get_block_info(arg0, arg1);
+        return  my->get_block_info(arguments, result);
     });
 }
 
 DEFINE_API ( plugin, get_blocks_with_info ) {
-    auto arg0 = args.args->at(0).as<uint32_t>();
-    auto arg1 = args.args->at(1).as<uint32_t>();
+    auto arguments = args.args->at(0).as<uint32_t>();
```
Comment (kotbegemot) [inline libraries/plugins/block_info/plugin.cpp]: start_block_num

File: libraries/plugins/block_info/plugin.cpp [hunk 3]
```diff
@@ -104,20 +104,20 @@ void plugin::plugin_impl::on_applied_block(const protocol::signed_block &b) {
...
+    auto result = args.args->at(1).as<uint32_t>();
```
Comment (kotbegemot) [inline libraries/plugins/block_info/plugin.cpp]: count

File: libraries/plugins/block_info/plugin.cpp [hunk 4]
```diff
@@ -104,20 +104,20 @@ void plugin::plugin_impl::on_applied_block(const protocol::signed_block &b) {
 }
 
 DEFINE_API ( plugin, get_block_info ) {
-    auto arg0 = args.args->at(0).as<uint32_t>();
-    auto arg1 = args.args->at(1).as<uint32_t>();
+    auto arguments = args.args->at(0).as<uint32_t>();
```
Comment (kotbegemot) [inline libraries/plugins/block_info/plugin.cpp]: start_block_num

File: libraries/plugins/block_info/plugin.cpp [hunk 5]
```diff
@@ -104,20 +104,20 @@ void plugin::plugin_impl::on_applied_block(const protocol::signed_block &b) {
...
+    auto result = args.args->at(1).as<uint32_t>();
```
Comment (kotbegemot) [inline libraries/plugins/block_info/plugin.cpp]: count

**Pull request merged**